docs/src/main/sphinx/release/release-360.md

```diff
@@ -0,0 +1,71 @@
+# Release 360 (XXX Jul 2021)
+
+## General
+
+* Reduce number of requests from coordinator to workers ({issue}`8460`)
```

Comment:
@sopel39 what does it mean from user perspective?

Reply:
`Reduce coordinator network load`?